service/history/consts/const.go

```diff
@@ -67,6 +67,8 @@ var (
 	ErrTransitionHistoryDisabled = serviceerror.NewFailedPrecondition("Transition history disabled")
 	// ErrActivityTaskNotFound is the error to indicate activity task could be duplicate and activity already completed
 	ErrActivityTaskNotFound = serviceerror.NewNotFound("invalid activityID or activity already timed out or invoking workflow is completed")
+	// ErrActivityNotFound is the error to indicate that there is no pending activity with this ID
+	ErrActivityNotFound = serviceerror.NewNotFound("Can't find pending activity with such ID. Invalid activityID or activity already completed")
```

Comment:
I think I reached some dzen in `ID` vs `Id` discussion. I can accept both.

Reply:
monkey see monkey do. In all other places it is "activityID" (one of them right above)